about_solid.html

```diff
@@ -54,8 +54,8 @@ <h1>Solid</h1>
                         <p class="pt-5">Solid is a specification that lets people store their data securely in
                             decentralized data stores
                             called Pods. Pods are like secure personal web servers for your data.</p>
-                        <p>You control access to the data in your Pod. You decide what data to share and with whom (be
-                            itindividuals, organizations, and/or applications). You can revoke access at any time.</p>
+                        <p>People control access to the data in their Pod. People decide what data to share and with whom (be
```

Comment:
I don't think this is limited to people. Organizations, businesses, and other entities have the same powers...
```suggestion
                        <p>Entities control access to the data in their Pod. Entities decide what data to share and with whom (be
```